### ISSUE TITLE
Make menu nicer

### ISSUE THREAD

**Issue by waaana** [ERNICommunity/ErniEventManager#25]
- animation for menu
- reanimate hambutger icon to x when clicked
- investigate & find a clever way how to display menu

**Comment by japutko** [comment]
https://www.w3schools.com/howto/howto_css_menu_icon.asp ---> hamburger animation

**Comment by rochus222** [comment]
do we even need hiding menu ? we have 3 different filters in menu (which should be in some filter controller in event list view -> maybe idea for next ticket), 1 create button (which does not make sense to have in menu) and users (which will be displayed for maybe 10 people that will have admin rights and therefore does not have that importance to be in separate menu... it can be easily added to footer or header)... in my opinion we do not have enough content for sliding menu we want to accomplish here

**Comment by srnec** [comment]
I'm with you in this, even if all the current buttons would be visible for all users it would be manageable to put them into the header. I think it was meant to be just a nice to have thing but I would also go with removal of this side menu

**Comment by japutko** [comment]
Yep, I totally agree. We're just overcomplicating design with it
